src/context/user/user-context.js

```diff
@@ -223,13 +223,13 @@ function userReducer(state, action) {
 
     case SET_CHECKOUT_ADDRESS: {
       // action is {type: SET_CHECKOUT_ADDRESS, payload: {address: {campus:string, building: string, floor_apartment: string}}}
```

Comment:
When you change the argument, make sure you update the comment. The comment here still show that the `payload` suppose to have `address` as a field